primitives/hp-proof-of-existence/src/lib.rs

```diff
@@ -19,14 +19,14 @@ use sp_core::H256;
 use sp_inherents::{InherentIdentifier, IsFatalError};
 
 /// The identifier for the `proof-of-existence0` inherent.
-pub const INHERENT_IDENTIFIER: InherentIdentifier = *b"PoE-0000";
+pub const INHERENT_IDENTIFIER: InherentIdentifier = *b"Att-0000";
```

Comment:
@giacgus this is the part causing the hard fork. If we want to avoid it, we can leave it as before without major issues I guess

Reply:
So impacts of changing this, being an inherent, are not clear. Worst case is a regenesis. Let's keep it as "PoE" for the moment and we will change it once we regenesis